app/views/admins/members/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= simple_form_for [:admin, @member] do |f| %>
+  <%= f.input :first_name, label: 'First name:', required: true %><br />
+  <%= f.input :last_name, label: 'Last name:', required: true %><br />
+  <%= f.input :email, label: 'Email:', required: true %><br />
```

Comment:
Do you have to put `required: true`? IIRC `simple_form` checks the model validations for required fields.